src/sentry/static/sentry/app/views/performance/trends/changedTransactions.tsx

```diff
@@ -506,24 +503,29 @@ const TransactionMenuButton = styled('button')`
 `;
 const TransactionMenuContainer = styled('div')`
   height: 100%;
-
   display: flex;
   justify-content: center;
   align-items: center;
 `;
 
-const TransactionsList = styled('div')``;
+const TransactionsList = styled('div')`
+  font-size: ${p => p.theme.fontSizeMedium};
+`;
+
 const ListItemContainer = styled('div')`
   display: flex;
   border-top: 1px solid ${p => p.theme.borderLight};
   padding: ${space(1)} ${space(2)};
 `;
 
 const ItemRadioContainer = styled('div')`
+  input {
+    cursor: pointer;
+  }
   input:checked::after {
     background-color: ${p => p.color};
-    width: 14px;
-    height: 14px;
+    width: 11px;
```

Comment:
So we want it to be `1px` larger than the default size? Is that a ❄️ at this point?

Reply:
Removed and reverted to default radio state :)